flink-connectors/flink-jdbc/src/main/java/org/apache/flink/api/java/io/jdbc/writer/AppendOnlyWriter.java

```diff
@@ -38,6 +40,7 @@
 	private final String insertSQL;
 	private final int[] fieldTypes;
 
+	private transient List<Row> cachedRows;
 	private transient PreparedStatement statement;
```

Comment:
Can you mock a `PreparedStatement` and add some unit tests?

Reply:
Could you give more tips? add some unit  tests for  checking/validating what.

Reply:
I mean you can add a unit test for `AppendOnlyWriter`.